hazelcast/src/main/java/com/hazelcast/config/WanTargetClusterConfig.java

```diff
@@ -24,6 +24,16 @@
 public class WanTargetClusterConfig {
 
     private static final WanAcknowledgeType DEFAULT_ACK_TYPE = WanAcknowledgeType.ACK_ON_OPERATION_COMPLETE;
+    private static final WanQueueFullBehavior QUEUE_FULL_BEHAVIOR = WanQueueFullBehavior.DISCARD_AFTER_MUTATION;
+    private static final int DEFAULT_BATCH_SIZE = 500;
+    private static final long DEFAULT_BATCH_MAX_DELAY_MILLIS = 1000;
+    private static final int DEFAULT_QUEUE_CAPACITY = 10000;
+    private static final long DEFAULT_RESPONSE_TIMEOUT_MILLIS = 60000;
```

Comment:
ok. I would prefer to create a constructor which is accepting all class fields and remove all setters. Default constructor would use those static fields. However, that is my preference...